internal/uvm/create.go

```diff
@@ -156,6 +178,12 @@ func verifyOptions(_ context.Context, options interface{}) error {
 		if opts.SCSIControllerCount != 1 {
 			return errors.New("exactly 1 SCSI controller is required for WCOW")
 		}
+		if err := verifyWCOWBootFiles(opts.BootFiles); err != nil {
```

Comment:
may also want to have a `nil` check here, just in case someone did something funky with the options